tests/robot-tests/tests/general_public/release_page_absence.robot

```diff
@@ -82,6 +82,7 @@ Validate subject files file type and file unit style
 Validate absence_in_prus.csv file can be downloaded
     [Documentation]    DFE-958    DFE-562
     [Tags]    NotAgainstLocal    Failing
+    # TODO: This needs data-guidance utils to be created to support the new way of downloading files
```

Comment:
I think a ticket ref would be good here, to tie together the commented-out code in get_auth_tokens.py with this comment, and to let another person who doesn't know the details understand what is needed from the data-guidance utils to support to new way of downloading, and what the new way of downloading is.

Reply:
Think this was a mistake. I was thinking of data guidance when I mentioned a 'new way of downloading files' (due to how the zip file gets dynamically generated). I'll update the comment to reflect what needs to be added.